docs/runbooks/HPPNotReady.md

```diff
@@ -2,31 +2,38 @@
 
 ## Meaning
 
-The hostpath-provisioner (HPP) is used to dynamically provision hostPath volumes to provide storage for PVCs.  
-The HPPNotReady alert is indicative of a HPP installation being in a degraded state;  
+The `hostpath-provisioner` (HPP) is used to dynamically provision hostPath volumes to provide storage for PVCs.  
+The HPPNotReady alert indicates that a HPP installation is in a degraded state:
+
 - Not progressing
+
 - Not available to use
 
 ## Impact
 
-HPP is simply not usable, its components are not ready & stopped progressing towards a ready state and hence require action.
+HPP is not usable. Its components are not ready and they stopped progressing towards a ready state.
```

Comment:
Suggestion: "The HPP is not usable. Its components are not ready and they are not progressing towards a `ready` state."